content/about/board/_index.md

```diff
@@ -81,3 +81,11 @@ Russ's drive and passion is to enable all software engineers to achieve incredib
 {{< board-member name="Maximilian Capraro" image="/images/about/Maximilian_Capraro.jpg" >}}
 Maximilian Capraro is a researcher and doctoral candidate at the Open Source Research Group at Friedrich-Alexander-University Erlangen-Nürnberg. He worked on inner source in research projects with a variety of companies including Black Duck Software, Continental, and multiple Siemens business units. He developed the patch-flow measurement method (for measuring software development collaboration) and the first classification framework for inner source projects and programs. His research interests the quantification of software development collaboration, inner source governance including legal and taxation aspects, and many other things around inner source, open source, and software engineering.
 {{< /board-member >}}
+
+{{< board-member name="Tim Yao" image="/images/about/Tim_Yao.jpeg" >}}
```

Comment:
Suggest using my portrait at https://avatars.githubusercontent.com/u/2311408?v=4

Reply:
Ah, nice one!
Updated in [6231379](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/278/commits/62313790451b5ffd7e46d02b6e63540497404ec3)